src/core/types/Yearn-Sdk.ts

```diff
@@ -59,6 +59,8 @@ interface LabMetadata {
   displayName: string;
   displayIcon: string;
   defaultDisplayToken: Address;
+  zapInWith?: string; // TODO: Import zapInType from sdk when updated
+  zapOutWith?: string; // TODO: Import zapOutType from sdk when updated
```

Comment:
Ahh now I see this TODO, we could also just use the same type I've set in there for now just in case 
```typescript
keyof Token["supported"]
```

Reply:
refactored here https://github.com/yearn/yearn-finance-v3/pull/674/commits/b70f2e7d0cd1238378dc4a49fa0fdee00c0bd66b